app/src/test/java/com/naxtlevelofandroiddevelopment/marvelgallery/CharacterPresenterUnitTest.kt

```diff
@@ -0,0 +1,77 @@
+@file:Suppress("IllegalIdentifier")
+
+package com.naxtlevelofandroiddevelopment.marvelgallery
+
+import com.naxtlevelofandroiddevelopment.marvelgallery.presenter.CharacterProfilePresenter
+import com.nextlevelofandroiddevelopment.marvelgallery.helpers.CharacterProfileViewHelper
+import com.nextlevelofandroiddevelopment.marvelgallery.helpers.Example.exampleCharacter
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class CharacterPresenterUnitTest {
+
+    @Test fun `Image Url is requested to be shown as an image`() {
+        var requestedImage: String? = null
+        val view = CharacterProfileViewHelper(
+                onSetUpCharacterImage = {
+                    requestedImage = it
+                }
+        )
+        val presenter = CharacterProfilePresenter(view, exampleCharacter)
+        presenter.onViewCreated()
+        assertEquals(exampleCharacter.imageUrl, requestedImage)
+    }
+
+    @Test fun `Correct user name and description displayed`() {
+        var displayedName: String? = null
+        var displayedDescription: String? = null
+        val view = CharacterProfileViewHelper(
```

Comment:
Helpers are just to allow Mocking? if yes then can be removed replaced with Mockito